src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Group.java

```diff
@@ -39,7 +39,7 @@
 
 	@NotNull
 	@Column(name = "group_number")
-	private long groupNumber;
+	private Long groupNumber;
```

Comment:
If it's not nullable, we should keep this on `long` instead of `Long`. Saves us from auto-boxing and potential NPEs (and checks).